<?xml version="1.0"?>
<launch>
    <arg name="game_config_dir_path" default="$(find migrave_behaviour_manager)/config/games" />
    <arg name="game_performance_topic" default="/migrave/game_performance" />

    <node pkg="migrave_games" type="migrave_game_colors" name="migrave_game_colors" output="screen" respawn="true">
        <param name="game_config_dir_path" type="str" value="$(arg game_config_dir_path)" />
        <param name="game_id" type="str" value="colors" />
        <param name="game_status_topic" type="str" value="/migrave_game_colors/status" />
        <param name="game_answer_topic" type="str" value="/migrave_game_colors/answer" />
        <param name="game_performance_topic" type="str" value="$(arg game_performance_topic)" />
    </node>

</launch>
